lib/include/openamp/virtio.h

```diff
@@ -182,7 +182,7 @@ struct _virtio_dispatch_ {
 	void (*write_config) (struct virtio_device * dev, uint32_t offset,
 			      void *src, int length);
 	void (*reset_device) (struct virtio_device * dev);
-
+	void (*notify) (struct virtqueue *vq);
```

Comment:
when is notify suppose to be called?

Reply:
this should not be in the virtio device, instead it should be in the virtqueue